src/settings/settingsReducer.js

```diff
@@ -14,6 +14,7 @@ const initialState: SettingsState = {
   onlineNotification: true,
   experimentalFeaturesEnabled: false,
   streamNotification: false,
+  browser: 'default',
```

Comment:
Let's add a migration for adding this to the settings state. I was a bit surprised to see things working, in my local testing, without one.

It turns out that the `REHYDRATE` action doesn't clobber the `initialState` with the state that's just been grabbed from storage. Rather, it does a "shallow merge": https://github.com/zulip/zulip-mobile/blob/v27.162/src/third/redux-persist/autoRehydrate.js#L69. That's why there's still a `state.settings.browser`, post-rehydration.

It took me a while staring at the code to find that fact, and I'm not aware that we intentionally depend on it anywhere. Also, `redux-persist` and `redux-persist-migrate` sort of work together, so this logic could plausibly change as we overhaul `redux-persist`.

We've also found that it's good to err on the side of being explicit and literal in what goes in migrations; e.g., in 1193a57cc346c9f2d684bcd6fee5bb0fa2d0ed11.

b058fa266881cf84570d6c9390c03eaabed15119 has an example of adding a migration for more data being stored. That one was definitely necessary, I think; the "shallow merge" couldn't have helped there.

Reply:
Done!